src/components/explain-sponsoring/explain-sponsoring.module.scss

```diff
@@ -0,0 +1,15 @@
+.root {
+  text-align: center;
+  max-width: 1000px;
+
+    li {
+        list-style: disc;
+    }
+}
+.text_left {
+    text-align: left;
+}
+
+.sub_headline {
+  margin: 26px 0;
+}
```

Comment:
Prettier / linter not complaining about the spaces? 😅

Reply:
not complaining. - but reminds me that formatting was not enabled.

I guess, we should once format all files properly, with the latest prettier config we have.

Reply:
I could do that ideally in a follow-up...

Reply:
I guess this is now okay, and let's ship this page!